index.js

```diff
@@ -0,0 +1 @@
+console.log('hello world')
```

Comment:
Let's add a semi-colon to conform to code style.

```suggestion
console.log('hello world');
```